fastlane/swift/Runner.swift

```diff
@@ -110,25 +153,39 @@ extension Runner {
 }
 
 extension Runner : SocketClientDelegateProtocol {
-    func commandExecuted(serverResponse: SocketClientResponse) {
+    func commandExecuted(serverResponse: SocketClientResponse, completion: (SocketClient) -> Void) {
         switch serverResponse {
         case .success(let returnedObject, let closureArgumentValue):
             verbose(message: "command executed")
             self.returnValue = returnedObject
             if let command = self.currentlyExecutingCommand as? RubyCommand {
-                if let closureArgumentValue = closureArgumentValue {
-                    command.performCallback(callbackArg: closureArgumentValue)
+                if let closureArgumentValue = closureArgumentValue, !closureArgumentValue.isEmpty {
+                    command.performCallback(callbackArg: closureArgumentValue, socket: socketClient) {
+                        print("Leave \(#function) \(#line)")
```

Comment:
Either delete or 👇 ?

```suggestion
                        verbose(message: "Leave \(#function) \(#line)")
```